### PR TITLE
Two Key Hash Map and Implementation

### PR DESCRIPTION
**Pull request by josephwhite**

### CODE REVIEW

File: src/Scene/Player.java [hunk 1]
```diff
@@ -138,7 +154,8 @@ else if (airGroundState == AirGroundState.AIR) {
             }
 
             if (moveAmountY < 0) {
-                currentAnimationName = facingDirection == Direction.RIGHT ? "JUMP_RIGHT" : "JUMP_LEFT";
+                //currentAnimationName = facingDirection == Direction.RIGHT ? "JUMP_RIGHT" : "JUMP_LEFT";
+                currentAnimationName = playerStateAndDirection.get(playerState.JUMPING, facingDirection);
             } else {
                 currentAnimationName = facingDirection == Direction.RIGHT ? "FALL_RIGHT" : "FALL_LEFT";
```
Comment (a-r-t) [inline src/Scene/Player.java:160]: Why no fall? :(

Comment (josephwhite) [reply src/Scene/Player.java:160]: No FALLING player state. :(

Comment (a-r-t) [reply src/Scene/Player.java:160]: I'm going to hold on this for a second while I finish something up, and then see if it makes sense to do a FALLING state and have one animation per state. I like this solution a lot though!!!

File: src/Utils/TwoKeyHashMap.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
+package Utils;
```
Comment (a-r-t) [inline src/Utils/TwoKeyHashMap.java:1]: This is neat, please site your source if you used any with a comment on top (take a look at my ImageUtils class for an example) -- want to show the students good behavior :)

**Comment by josephwhite** [comment]
Editing some things to make them more compat. to recent changes.

**Pull request closed without merging**